frontend/src/Breadcrumb.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+    Breadcrumbs,
+    Typography
+} from "@material-ui/core"
+import parse from 'html-react-parser';
+
+function Breadcrumb() {
+
+    var element = "<Breadcrumbs aria-label=\"breadcrumb\">";
+    var currentBreadcrumb = "/"
+
+    for (var i = 0; i < arguments.length; i++) {
+        if (i == (arguments.length - 1)) {
+            element.concat("<Typography color=\"textPrimary\">", arguments[i], "</Typography>")
+        }
+        currentBreadcrumb.concat("/", arguments[i]);
+
+        element += "<Link color=\"inherit\" href=\""
+            + currentBreadcrumb + "\" onClick={handleClick}>"
+            + arguments[i] + "</Link>);"
+    }
+
+    element.concat("</Breadcrumbs>")
+    return parse(element)
+}
+
+class Breadcrumb extends React.Component {
```

Comment:
I made it to build the breadcrumb dynamically, instead of hard coding it. But it was not used. I think it would be good to have it dynamically set up, but I don't know shit all about how props work yet so that's pending